gui/notesView.py

```diff
@@ -0,0 +1,41 @@
+import wx
+
+import service
+import gui.globalEvents as GE
+import gui.display as d
+import gui.mainFrame
+
+class NotesView(d.Display):
+    DEFAULT_COLS = []
```

Comment:
The problem here is that you're Inheriting `d.Display`. That is what we use for our lists (fitting view, cargo view, etc... anywhere that is a list of item with informational columns (I guess table is a better term for this lol)). What you want to do is something like the current fleet view, where you inherit wx.Panel (IIRC) and build a simple page out of it that includes a textbox